python/foglamp/plugins/south/playback/playback.py

```diff
@@ -0,0 +1,453 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for playback async plugin """
+
+import asyncio
+import copy
+import csv
+import uuid
+import os
+import json
+import logging
+import datetime
+import time
+from threading import Event
+from dateutil import parser
+
+from queue import Queue
+from threading import Thread, Condition
+
+from foglamp.common import logger
+from foglamp.plugins.common import utils
+from foglamp.services.south import exceptions
+from foglamp.services.south.ingest import Ingest
+
+
+__author__ = "Amarendra Kumar Sinha"
+__copyright__ = "Copyright (c) 2018 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+        'description': 'Test Sample',
+        'type': 'string',
+        'default': 'playback',
+        'readonly': 'true'
+    },
+    'assetName': {
+        'description': 'Name of Asset',
+        'type': 'string',
+        'default': 'sample',
+        'displayName': 'Asset Name',
+        'order': '1'
+    },
+    'csvFilename': {
```

Comment:
we need to put this CSV file manually?

Reply:
Currently this will be fine. We could change to use the script type of configuration item and allow the file to be uploaded.